scripts/pre_commit_linter.py

```diff
@@ -2178,6 +2192,30 @@ def _check_codeowner_file(self):
                             print ('WARNING! %s/%s is not covered under '
                                    'CODEOWNERS' % (root, file_name))
 
+            # Checks that the most important rules/patterns are at the bottom
```

Comment:
I think to check code owner function is already 150~ lines big can we create a sperate function for checking important rule or pattern and call this over here? will it be possible?   

This will also move this many comments into the new function docstring!

Reply:
Yeah sounds good. Will do that!
Also, @apb7, do you agree on this too? (since you are always in favor of clubbing similar functionalities to a pre-existing function if any)

Reply:
I agree with @DubeySandeep here. We can modularize this code. (As an FYI -- I am in favor of clubbing similar checks so that we do not end up repeating stuff. If there's no repetition, then it's totally fine :))

Reply:
@YashJipkate, is this comment resolved?

Reply:
Yes @DubeySandeep This is resolved now. Sorry for the delay.
PTAL! Thanks